country.html

```diff
@@ -51,6 +51,17 @@
       </nav>
     </header>
     <main class="uk-padding uk-flex-column">
+      <!-- Country Name Card -->
+      <section
+        class="card uk-flex-column uk-box-shadow-larges uk-padding-small uk-margin text-white"
```

Comment:
uk-box-shadow-large not larges.  you can just leave this off my last pull adds box shadow